packages/vscode-extension/package.json

```diff
@@ -1032,6 +1038,22 @@
                   }
                 },
                 "additionalProperties": false
+              },
+              {
+                "type": "object",
+                "required": [
+                  "configFile",
```

Comment:
I prefer `template` to just be consistent with the CLI command.

Reply:
Updated.